org.eclipse.winery.repository.ui/src/app/wineryInterfaces/enums.ts

```diff
@@ -45,3 +45,12 @@ export enum ServiceTemplateTemplateTypes {
     RelationshipTemplate = 'relationshipTemplates',
     RequirementTemplate = 'requirementTemplates'
 }
+
+/**
+ * BackendAvailabilityStates defines the states for the availability of the backend.
```

Comment:
Normal comments for a enum, class, ... do not state the name of the enum, class, ... again. I'll merge nevertheless. This will be fixed in master.